### DISCUSSION TITLE
怎么部署到cf worker上啊？

### DISCUSSION THREAD

**Discussion by fengzhao** [spencerwooo/onedrive-vercel-index#96] [Q&A] [answered]
怎么部署到cf worker上啊？

**Comment by spencerwooo** [comment]
为什么要部署到 CF worker 上？

**Reply by fengzhao** [reply]
感觉cf比vercel速度应该更好

**Comment by myl7** [comment]
现在的 CF Pages 还没有支持 Next 的 SSR，只有 SSG，而这个项目 /api 就用了 SSR，所以在 CF Pages 支持 Next SSR 之前应该都没法把这部署在 CF 上
或许可以在 CF Workers 里自己处理 SSR 部分的逻辑，但要么麻烦要么改一堆，总之不太方便

**Comment by spencerwooo** [comment, marked as answer]
暂时不会考虑除 Vercel 和 Netlify 外的其他平台。
